docs/release-notes/mia-platform-v13.mdx

```diff
@@ -7,3 +7,141 @@ image: "img/release-note-link-preview.png"
+Let's take a closer look at all the new features included in Mia-Platform v13.
+
+## Platforge: Enhance Extensibility
+
+In today's fast-paced tech landscape, adaptability is key. That's why Mia-Platform v13 takes extensibility to the next level, making it easier than ever to integrate third-party tools and custom functionalities into your platform. We welcome **Platforge**, our brand new Extensibility toolkit, in the Mia-Platform eco-system!
+The main benefits for the user can be summarized by the following:
+
+- Extend developer lifecycle to be aligned with the needs of each team (Extend supported runtime, extend dev workflow, extend application management…).
+- Brownfield can standardize and simplify their developer workflow.
+
+Let's go in deep with the new Features!
+
+### Extend Frontend of Mia-Platform Console with iFrames
+
+Extend the Console as you like.
+
+From now on, you can seamlessly integrate external web pages within the Mia-Platform Console using iFrames. 
+
+This empowers Console Users to enrich their experience by effortlessly incorporating tools, dashboards, or external content in our IDP, streamlining development workflows and tailoring the product to suit the business.  
+
+INSERT AN IMAGE -> mettiamo immagine della sidebar con scritto my extension??????
+
+To learn more, please refer to the [documentation]().
+
+INSERT THE WORKING LINK
+
+### React to Console actions using Webhooks
+
+The variety of Console events suitable for webhook configuration constantly increases, allowing you to react and build custom flows and logic each time the event occurs. 
+
+With the introduction of user management events and the introduction of the history of each webhook to facilitate monitoring and debugging, users can now use webhooks to extend Console functionality.
+
+INSERT AN IMAGE -> mettiamo la history dei webhook!
+
+To learn more, please refer to the [documentation](/development_suite/webhooks-and-events/webhooks.md#event-history).
+
+### Connect Backstage portal to Mia-Platform
+
+On Backstage Marketplace you will find the Mia-Platform plugin for integrating Mia-Platform components into Backstage components.  
+It keeps your information synchronized effortlessly and gains insights into your Company's domains, Projects, deployment environments, and repository links, all while exploring their interrelations.
+
+## Simplify Composability
+
+Mia-Platform Suite empowers development teams to enhance software composability by facilitating streamlined reuse and embracing scalable approaches. With Mia-Platform v13, software composability is even more NoCode! 
+The main benefits for the user can be summarized by the following:
+
+- Democratizing access to the IDP's internal tools and utilities
```

Comment:
Here I might add that these tools are NO/LOW Code evolutions of their predecessors, which enhance the Developer Experience (rather than just being "user friendly")